notebooks/Integrate_NLP_Libraries.ipynb

```diff
@@ -174,7 +174,7 @@
     {
```

Comment:
<p>'along' changes fro <code>ADV</code>  -&gt; <code>ADP</code>  , my grammar isn't great - is that wrong now?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/CODAIT/text-extensions-for-pandas/pull/169/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='49'/>

Reply:
<p>I believe that <code>ADP</code>  is the right answer here. The tagging system in NLU is based on Universal Dependencies tags iirc; see http://universaldependencies.org/docs/en/pos/all.html</p>